Formula/snapcast.rb

```diff
@@ -0,0 +1,51 @@
+class Snapcast < Formula
+  desc "Synchronous multi-room audio player"
+  homepage "https://github.com/badaix/snapcast"
+  url "https://github.com/badaix/snapcast/archive/v0.10.0.tar.gz"
+  version "0.0.10"
+  sha256 "e4a9e12aca8cfeed5e51866a0143e7e8d0a6e1ce0c076e8f78ce23c5c735c8d3"
+  revision 1
+  head do
+    url "https://github.com/badaix/snapcast.git"
+  end
+  depends_on "flac"
+  depends_on "libvorbis"
+  def install
+    args = %W[
+      prefix=#{prefix}
+      sysconfdir=#{etc}
+      CXX=#{ENV.cxx}
+    ]
```

Comment:
Please inline the arguments instead of creating an array; it’s not necessary here.